msteams-platform/tabs/what-are-tabs.md

```diff
@@ -130,7 +130,9 @@ A custom tab is declared in the app manifest of your app package. For each webpa
 Whether you choose to expose your tab within the channel or group, or personal scope, you must present an <iframe\> HTML [content page](~/tabs/how-to/create-tab-pages/content-page.md) in your tab. For static tabs, the content URL is set directly in your Teams [app manifest](../resources/schema/manifest-schema.md#statictabs) by the `contentUrl` property in the `staticTabs` array. Your tab's content is the same for all users.
 
 > [!Note]
-> Teams app doesn't recognize sub iframes. Therefore, it'll not load if there is an iframe within the tab app.
+>
+> * Teams app doesn't support sub iframes. Therefore, it'll not load if there is an iframe within the tab app.
```

Comment:
This note is wrong and needs to be deleted. 

![image](https://github.com/MicrosoftDocs/msteams-docs/assets/31851992/bee2cab3-41e0-48bb-a97d-f1bf6ca8c070)

Reply:
Removed the note as suggested.